README.md

```diff
@@ -70,6 +70,12 @@ and the [SDL2 library](https://www.libsdl.org/).
 * macOS: `brew install sdl2 jpeg libpng`
 * Ubuntu Linux / Debian: `sudo apt install libsdl2-dev libjpeg-dev libpng-dev`
 
+For those who want to run demo program with VNC as the backend, it is necessary to install the [aml](https://github.com/any1/aml) and [neatvnc](https://github.com/any1/neatvnc) libraries. Please note that using package managers like apt might install outdated versions of these libraries. To ensure you have the latest versions, you can build them from source by running the provided script: 
+
+```bash
+$sudo ./tools/build-neatvnc.sh
```

Comment:
Don't run with `sudo`.

Reply:
Since `ninja -C build install` requires root access, may I know the reason for removing sudo here?

Reply:
The way to pass an option to Meson is using the `-D` option. So to set the prefix, you should use `meson -Dprefix=$(PWD)/_statatic build`.

Reply:
No, the primary purpose of this build script is to simplify the installation of build-time dependencies. While users can manually handle the dependencies themselves, the script provides a convenient alternative. We specifically chose to install the dependencies as statically-linked libraries to minimize required system modifications, ensuring the least intrusive setup process possible.